app/geant-exporter/geant-exporter.cc

```diff
@@ -26,19 +26,24 @@
 #include <TTree.h>
 #include <TBranch.h>
 
-#include "ActionInitialization.hh"
-#include "DetectorConstruction.hh"
-#include "PhysicsList.hh"
-#include "GeantPhysicsTableWriter.hh"
+#include "base/Range.hh"
 #include "comm/Communicator.hh"
 #include "comm/Logger.hh"
 #include "comm/ScopedMpiInit.hh"
 #include "io/ImportParticle.hh"
 #include "io/ImportPhysicsTable.hh"
 #include "io/GdmlGeometryMap.hh"
-#include "base/Range.hh"
+#include "physics/base/PDGNumber.hh"
+
+#include "ActionInitialization.hh"
+#include "DetectorConstruction.hh"
```

Comment:
Are those 2 headers implied by some other headers (i.e. the new code those not seem to use them).

Reply:
I just rearranged the headers so that "local source directory" headers are after "library" headers. The file does use them.